benchmark/python/sparse/sparse_end2end.py

```diff
@@ -57,13 +57,15 @@
 parser.add_argument('--measure-only', default=None,
                     help="Measure only",
                     choices=[IO, COMP, COMM])
+parser.add_argument('--omit-row-sparse-push', action='store_true',
+                    help="omit row_sparse_push")
 
 
 def get_libsvm_data(data_dir, data_name, url, data_origin_name):
     if not os.path.isdir(data_dir):
         os.system("mkdir " + data_dir)
     os.chdir(data_dir)
-    if (not os.path.exists(data_name)):
+    if not os.path.exists(data_name):
         import urllib
         zippath = os.path.join(data_dir, data_origin_name)
         urllib.urlretrieve(url, zippath)
```

Comment:
Use bz2 instead of relying on the bzip2 command in the os. https://docs.python.org/2/library/bz2.html

Reply:
for portability

Reply:
Moved the get_bz2_data to test_utils and reused it in tests and benchmarking scripts.